Exercises/if-else-switch/if-else-switch.js

```diff
@@ -6,9 +11,36 @@ const codeToMissingComplexityElement = {
 };
 
 function getMissingComplexityElementCode(password) {
-
+  if (!password) {
+    return getErrorMessage(3);
+  }
+  if (password.length < 10) {
+    return getErrorMessage(0);
+  }
+  if (password.toLowerCase() === password) {
+    return getErrorMessage(1);
+  }
+  if (!password.match(/\W+/)) {
+    return getErrorMessage[2];
+  }
+  return 'Success';
 }
-
-function getErrorMessage() {
-  return false;
+function getErrorMessage(errorCode) {
+  switch (errorCode) {
```

Comment:
here you need to take by code missing element and depends on it generate message